sdk/include/reactos/undocshell.h

```diff
@@ -684,6 +684,8 @@ LargeIntegerToString(
     _In_opt_ const NUMBERFMTW *pNumberFormat,
     _In_ DWORD dwNumberFlags);
 
+LPWSTR WINAPI ShortSizeFormatW(_In_ DWORD dwNumber, _Out_writes_z_(0x8FFF) LPWSTR pszBuffer);
```

Comment:
Please use multi-line formatting, like the function prototype that is above.

Reply:
Fixed.